apidoc/Facebook.yml

```diff
@@ -488,6 +488,11 @@ methods:
         summary: |
             Currency, is denoted as, e.g. "USD", "EUR", "GBP".  See ISO-4217 for specific values.
         type: String
+      - name: parameters
+        summary: |
+            A dictionary object containing optional parameters (since 8.2.0).
```

Comment:
Why not add a since: "8.2.0" entry here?

Reply:
Also, we seem to be mixing SDK version numbers in here while this is the module API. I would assume we'd refer to the module's versions in these apidocs.

Reply:
> Why not add a since: "8.2.0" entry here?

'since' is not available for specific parameters of method.
After changing it I was getting error - 
`[ERROR] apidoc/Facebook.yml
Modules.Facebook: found 1 error(s)!
	methods
		logPurchase
			parameters
				parameters
					Invalid key(s) in Modules.Facebook: since`